dts/nios2/nios2f.dtsi

```diff
@@ -38,5 +38,14 @@
 			status = "disabled";
 		};
 
+		i2c0: i2c@100200 {
+			compatible = "nios2,i2c";
+			clock-frequency = <50000000>;
```

Comment:
Add an include:
`#include <dt-bindings/i2c/i2c.h>`

And change this to:
`clock-frequency = <I2C_BITRATE_ULTRA>;`

Reply:
done